packages/common/package.json

```diff
@@ -42,7 +42,7 @@
     "bignumber.js": "^8.0.1",
     "chalk-pipe": "^3.0.0",
     "dotenv": "^6.2.0",
-    "eth-crypto": "^1.6.0",
+    "eth-crypto": "^1.7.0",
```

Comment:
not sure if this is strictly necessary, but running locally without this change doesnt seem to be an option